src/server/services/vergunningen.ts

```diff
@@ -314,19 +353,8 @@ export function createVergunningNotification(
     const allItems = items.filter(
```

Comment:
Is dit nog valide? Hebben de nieuwe vergunningen een afloopdatum?

Reply:
Shit ik had deze functie generiek gemaakt, maar lijkt met de merge uit main, te zijn verdwenen. Ga ik even fixen.